cmake/DaemonNacl.cmake

```diff
@@ -25,21 +25,35 @@
 # SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 # Native client
-if( APPLE )
-  add_definitions( -DNACL_WINDOWS=0 -DNACL_LINUX=0 -DNACL_ANDROID=0 -DNACL_OSX=1 )
-elseif( LINUX )
-  add_definitions( -DNACL_WINDOWS=0 -DNACL_LINUX=1 -DNACL_ANDROID=0 -DNACL_OSX=0 )
-elseif( WIN32 )
-  add_definitions( -DNACL_WINDOWS=1 -DNACL_LINUX=0 -DNACL_ANDROID=0 -DNACL_OSX=0 )
-endif()
-if( ARCH STREQUAL "x86" OR ARCH STREQUAL "x86_64" )
+
+if( ARCH STREQUAL "pnacl" )
```

Comment:
Usually we write `if (NACL)` for this condition

Reply:
Well, the rest of the code compares with `ARCH`, but whatever, both work.